main.go

```diff
@@ -33,16 +39,21 @@ func main() {
 
 	r := prometheus.NewRegistry()
 	r.MustRegister(httpRequestsTotal)
+	r.MustRegister(httpRequestDuration)
 	r.MustRegister(version)
 
-	handler := http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+	found := http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		histogram := httpRequestDuration.WithLabelValues("200", "found")
```

Comment:
Either we need to specify which specific 200 it is, or we change this to "2xx".

Reply:
Following your suggestion, I used the `InstrumentHandlerDuration` function instead of doing the measurement manually. Hence, we don't have a hardcoded value for the status anymore.